.flowconfig

```diff
@@ -5,6 +5,7 @@
 # Tracking https://github.com/facebook/flow/issues/4015
 <PROJECT_ROOT>/docs
 <PROJECT_ROOT>/test
+<PROJECT_ROOT>/packages/gestalt-codemods/.*/__testfixtures__
```

Comment:
Flow fails on `<Text bold />` referenced in test fixtures, we could add something like `$FlowExpectedError`, however that breaks the `input` / `output` behavior `defineTest` expects